examples/dt/bgp/dataplane.md

```diff
@@ -0,0 +1,37 @@
+# Configuring and deploying the dataplane
+
+## Assumptions
+
+- The [control plane](control-plane.md) has been created and successfully deployed
+- An infrastructure of spine/leaf routers exists, is properly connected to the
+  pre-provisioned EDPM nodes and the routers are configured to support BGP.
+
+## Initialize
+
+Switch to the "openstack" namespace
+```
+oc project openstack
+```
+Change to the bgp/edpm directory
+```
+cd architecture/examples/dt/bgp/edpm
+```
+Edit the [values.yaml](edpm/values.yaml) file to suit 
+your environment.
+```
+vi values.yaml
+```
+Generate the dataplane CRs.
+```
+kustomize build > dataplane.yaml
```

Comment:
After rebasing on the main branch, this command will need to be updated since it will fail as below. Please split the dataplane nodeset and deployment into separate CR files.

```
[fultonj@runcible edpm{bgp-basic-config}]$ kustomize build > dataplane.yaml
Error: accumulating components: accumulateDirectory: "recursed accumulation of path '/home/fultonj/review/architecture/dt/bgp/edpm': accumulating components: accumulateDirectory: \"recursed accumulation of path '/home/fultonj/review/architecture/lib/dataplane': accumulating components: accumulateDirectory: \\\"recursed accumulation of path '/home/fultonj/review/architecture/lib/dataplane/nodeset': nothing selected by ConfigMap.[noVer].[noGrp]/edpm-nodeset-values.[noNs]:data.nodeset.ansible\\\"\""
[fultonj@runcible edpm{bgp-basic-config}]$
```

Reply:
I have provided a PR against the base repo/branch to implement the refactoring. Please give it a check if you have a chance:  https://github.com/eduolivares/architecture/pull/1